app/experimenter/experiments/serializers/entities.py

```diff
@@ -182,6 +205,7 @@ class ExperimentSerializer(serializers.ModelSerializer):
     countries = CountrySerializer(many=True)
     pref_type = PrefTypeField()
     changes = ExperimentChangeLogSerializer(many=True)
+    results = serializers.SerializerMethodField()
```

Comment:
I wish there was a way to do this as `ResultsSerializer(source='self')` so you don't need the method